build_library/prod_image_util.sh

```diff
@@ -101,7 +102,8 @@ create_prod_image() {
           || die_notrace "coreos-au-key is missing the 'official' use flag"
   fi
 
-  tar -cf "${BUILD_DIR}/${image_pkgdb}" -C "${root_fs_dir}" var/cache/edb var/db/pkg
```

Comment:
I suggest we keep this line for the time being - it's used for the current Azure OEM sysext build.
While keeping the line adds redundancy, removing it would require us us to also update the Azure OEM sysext build script to use `build_sysext` instead of the system it currently uses.
Keeping the line would enable us to update the Azure OEM sysext build in a follow-up PR instead.

Reply:
It's gone again